corehq/apps/linked_domain/README.md

```diff
@@ -1,9 +1,65 @@
 # Linking Domains
 
-* Ability to sync models between two domains (master domain -> linked domain)
-* Can be within the same HQ instance or between remote instances.
-
-# Remote linking
+Linked project spaces allow project spaces to share much of their data. You configure a controlling upstream domain
+(called "master" in most code, though we are replacing that over time) and one or more downstream domains
+(called "linked" in most code, similarly being replaced). You can then overwrite downstream content with upstream content.
```

Comment:
nit: I think this would be slightly clearer to read if you pulled the parentheticals out to the end of the paragraph.